src/model/core/co2.jl

```diff
@@ -69,21 +69,22 @@ function co2!(EP::Model, inputs::Dict)
 
     ### Expressions ###
     # CO2 emissions from power plants in "Generators_data.csv"
-    # If all the CO2 capture fractions from Generators_data are zeros, the CO2 emissions from thermal generators are determined by fuel consumption times CO2 content per MMBTU 
+    # If all the CO2 capture fractions from Generators_data are zeros, the CO2 emissions from thermal generators are determined by fuel consumption times CO2 content per MMBTU
 
     if isempty(CCS)
         @expression(EP, eEmissionsByPlant[y = 1:G, t = 1:T],
             if y in SINGLE_FUEL
-                ((1 - biomass(gen[y])) * (EP[:vFuel][y, t] + EP[:vStartFuel][y, t]) *
-                 fuel_CO2[fuel(gen[y])])
+                (1 - biomass(gen[y])) * fuel_CO2[fuel(gen[y])] * (EP[:vFuel][y, t] + EP[:vStartFuel][y, t])
```

Comment:
This is a weird thing that probably doesn't make a difference, but it's worth explaining.

JuMP processes from left to right.

You have `a::Float64 * b::AffExpr * c::Float64`. JuMP rewrites as something like
```julia
d::AffExpr = a * b
d = map_coefficients!(d, c)
```
If you swap to `a::Float64 * c::Float64 * b::AffExpr`, then JuMP will do:
```julia
d::Float64 = a * c
e::AffExpr = d * c
```
This saves `N-1` multiplications where `N` is the number of coefficients (in this case, just two(?)), but you get the general idea.

Reply:
Thank you for clearing this up!